pydrive2/auth.py

```diff
@@ -521,12 +521,14 @@ def Authorize(self):
 
     :raises: AuthenticationError
     """
+    if self.access_token_expired and self.auth_method != 'service':
```

Comment:
so, we don't need this additional check anymore? service credentials do have `access token` and it should not be expired at this point unless I'm missing something. There should no be difference between workflows in this case.

Reply:
right, makes sense to remove this check